### DISCUSSION TITLE
Welcome to pixelfed Discussions!

### DISCUSSION THREAD

**Discussion by dansup** [pixelfed/pixelfed#4583] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cogdog** [comment]
Someone has to reply first, hello Daniel and thanks for pixelfed. I'm a [long time flickr user](https://www.flickr.com/photos/cogdog/) (since it started in 2004) with 70k photos all open licensed. I work in higher education on creative use of the web and also as an advocate for open sharing, some bits of tech development but mostly as a tinkerer (HTML/CSS/JS, PHP, WordPress), especially on the benefits of attributing media (I love how thats built in to Pixelfed to just copy aa full attribution text)

I've been working hard to convince people to embrace the fediverse (started on Mastodon in 2016) and have recently been intrigued to find the connectivity you've created here for photos (e.g. being able to follow photo feeds in Mastodon).

I've just started at https://pixelfed.social/cogdog 

I likely do not have much to contribute to code, but will share ideas/request/suggestions from the user side (if thats wanted here).

Thanks again for making possible federating of photos.
